diff --git a/node_modules/@celo/utils/lib/sign-typed-data-utils.js b/node_modules/@celo/utils/lib/sign-typed-data-utils.js
index 917066f..4571f45 100644
--- a/node_modules/@celo/utils/lib/sign-typed-data-utils.js
+++ b/node_modules/@celo/utils/lib/sign-typed-data-utils.js
@@ -81,14 +81,19 @@ var t = __importStar(require("io-ts"));
 var web3_eth_abi_1 = __importDefault(require("web3-eth-abi"));
 /** Array of all EIP-712 atomic type names. */
 exports.EIP712_ATOMIC_TYPES = [
-    'bytes1',
-    'bytes32',
-    'uint8',
-    'uint64',
-    'uint256',
-    // This list should technically include all types from uint8 to uint256, and int8 to int256
-    'int8',
-    'int256',
+    // bytes types from 1 to 32 bytes
+    // and uint/int types from 8 to 256 bits
+    // TODO: remove this patch once
+    // https://github.com/celo-org/celo-monorepo/pull/10224 is released
+    ...(function () {
+      const result = []
+      for (let i = 1; i <= 32; i++) {
+        result.push('bytes' + i)
+        result.push('uint' + i * 8)
+        result.push('int' + i * 8)
+      }
+      return result
+    })(),
     'bool',
     'address',
 ];
