diff --git a/node_modules/@react-native-google-signin/google-signin/RNGoogleSignin.podspec b/node_modules/@react-native-google-signin/google-signin/RNGoogleSignin.podspec
index 2d4d672..dc70716 100644
--- a/node_modules/@react-native-google-signin/google-signin/RNGoogleSignin.podspec
+++ b/node_modules/@react-native-google-signin/google-signin/RNGoogleSignin.podspec
@@ -16,5 +16,5 @@ Pod::Spec.new do |s|
   s.source_files = "ios/**/*.{h,m,mm}"
 
   s.dependency "React-Core"
-  s.dependency "GoogleSignIn", "~> 6.2"
+  s.dependency "GoogleSignIn", "~> 7.0"
 end
