diff --git a/node_modules/@react-navigation/stack/src/views/Header/HeaderContainer.tsx b/node_modules/@react-navigation/stack/src/views/Header/HeaderContainer.tsx
index ea19fab..254b426 100644
--- a/node_modules/@react-navigation/stack/src/views/Header/HeaderContainer.tsx
+++ b/node_modules/@react-navigation/stack/src/views/Header/HeaderContainer.tsx
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Animated, View, StyleSheet, StyleProp, ViewStyle } from 'react-native';
+import { View, StyleSheet, StyleProp, ViewStyle } from 'react-native';
 import {
   NavigationContext,
   NavigationRouteContext,
@@ -58,7 +58,7 @@ export default function HeaderContainer({
   const parentPreviousScene = React.useContext(PreviousSceneContext);
 
   return (
-    <Animated.View pointerEvents="box-none" style={style}>
+    <View pointerEvents="box-none" style={style}>
       {scenes.slice(-3).map((scene, i, self) => {
         if ((mode === 'screen' && i !== self.length - 1) || !scene) {
           return null;
@@ -153,7 +153,7 @@ export default function HeaderContainer({
           </NavigationContext.Provider>
         );
       })}
-    </Animated.View>
+    </View>
   );
 }
 
