diff --git a/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts b/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
index 961c442..aba4c3a 100644
--- a/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
+++ b/node_modules/@segment/sovran-react-native/lib/typescript/src/index.d.ts
@@ -1,3 +1,6 @@
-export { createStore, type Store, type Notify, type Unsubscribe, } from './store';
 export { registerBridgeStore } from './bridge';
 export * from './persistor';
+export { createStore, Notify, Store, Unsubscribe } from './store';
+ // TODO: This file can be removed when we upgrade to Typescript 4.4.X or higher.
+ // It was causing an error similar to this with our current typescript version segmentio/analytics-react-native#607
+
