diff --git a/node_modules/@segment/analytics-react-native-clevertap/android/build.gradle b/node_modules/@segment/analytics-react-native-clevertap/android/build.gradle
index 6d4ab82..bae8f27 100644
--- a/node_modules/@segment/analytics-react-native-clevertap/android/build.gradle
+++ b/node_modules/@segment/analytics-react-native-clevertap/android/build.gradle
@@ -8,7 +8,7 @@ buildscript {
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:3.1.4'
-        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21'
+        classpath 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.5.21'
     }
 }
 
