diff --git a/node_modules/clevertap-react-native/clevertap-react-native.podspec b/node_modules/clevertap-react-native/clevertap-react-native.podspec
index c27cf1f..dea6876 100644
--- a/node_modules/clevertap-react-native/clevertap-react-native.podspec
+++ b/node_modules/clevertap-react-native/clevertap-react-native.podspec
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.source_files   = 'ios/CleverTapReact/*.{h,m}'
 
   s.dependency 'CleverTap-iOS-SDK', '3.9.3'
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
