diff --git a/node_modules/clevertap-react-native/clevertap-react-native.podspec b/node_modules/clevertap-react-native/clevertap-react-native.podspec
index c27cf1f..dea6876 100644
--- a/node_modules/clevertap-react-native/clevertap-react-native.podspec
+++ b/node_modules/clevertap-react-native/clevertap-react-native.podspec
@@ -19,5 +19,5 @@ Pod::Spec.new do |s|
   s.source_files   = 'ios/CleverTapReact/*.{h,m}'
 
   s.dependency 'CleverTap-iOS-SDK', '3.9.3'
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
diff --git a/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTap+InAppNotifications.h b/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTap+InAppNotifications.h
new file mode 100644
index 0000000..11dc85c
--- /dev/null
+++ b/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTap+InAppNotifications.h
@@ -0,0 +1,37 @@
+#import <Foundation/Foundation.h>
+#import "CleverTap.h"
+
+@interface CleverTap (InAppNotifications)
+
+#if !CLEVERTAP_NO_INAPP_SUPPORT
+/*!
+ @method
+ 
+ @abstract
+ Suspends and saves inApp notifications until 'resumeInAppNotifications' is called for current session.
+ 
+ Automatically resumes InApp notifications display on CleverTap shared instance creation. Pending inApp notifications are displayed only for current session.
+ */
+- (void)suspendInAppNotifications;
+
+/*!
+ @method
+ 
+ @abstract
+ Discards inApp notifications until 'resumeInAppNotifications' is called for current session.
+ 
+ Automatically resumes InApp notifications display on CleverTap shared instance creation. Pending inApp notifications are not displayed.
+ */
+- (void)discardInAppNotifications;
+
+/*!
+ @method
+ 
+ @abstract
+ Resumes displaying inApps notifications and shows pending inApp notifications if any.
+ */
+- (void)resumeInAppNotifications;
+
+#endif
+
+@end
diff --git a/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTapURLDelegate.h b/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTapURLDelegate.h
new file mode 100644
index 0000000..af59f7b
--- /dev/null
+++ b/node_modules/clevertap-react-native/ios/CleverTapReact/CleverTapSDK.framework/Headers/CleverTapURLDelegate.h
@@ -0,0 +1,25 @@
+#import <Foundation/Foundation.h>
+
+@protocol CleverTapURLDelegate <NSObject>
+
+/*!
+ @method
+ 
+ @abstract
+ Implement custom handling of URLs.
+ 
+ @discussion
+ Use this method if you would like to implement custom handling for URLs in case of in-app notification CTAs, push notifications and App inbox.
+ 
+ Use the following enum values of type CleverTapChannel to optionally implement URL handling based on the corresponding CleverTap channel.
+ 
+ CleverTapPushNotification - Remote Notifications,
+ CleverTapAppInbox - App Inbox,
+ CleverTapInAppNotification - In-App Notification
+ 
+ @param url                     the NSURL object
+ @param channel            the CleverTapChannel enum value
+ */
+- (BOOL)shouldHandleCleverTapURL:(NSURL *_Nullable )url forChannel:(CleverTapChannel)channel;
+
+@end
