diff --git a/node_modules/react-native-fast-crypto/android/build.gradle b/node_modules/react-native-fast-crypto/android/build.gradle
index dd999fa..79db185 100644
--- a/node_modules/react-native-fast-crypto/android/build.gradle
+++ b/node_modules/react-native-fast-crypto/android/build.gradle
@@ -47,15 +47,6 @@ android {
             path "src/main/cpp/CMakeLists.txt"
         }
     }
-
-    // If you want Gradle to package prebuilt native libraries
-    // with your APK, modify the default source set configuration
-    // to include the directory of your prebuilt .so files as follows.
-    sourceSets {
-        main {
-            jniLibs.srcDirs 'jni/libs/'
-        }
-    }
 }
 
 repositories {
diff --git a/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec b/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
index 0745247..ce5333d 100644
--- a/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
+++ b/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
@@ -21,6 +21,6 @@ Pod::Spec.new do |s|
     "ios/Libraries/libnativecrypto.a",
     "ios/Libraries/libsecp256k1.a"
 
-  s.dependency "React"
+  s.dependency "React-Core"
   s.dependency "OpenSSL-Universal"
 end
