diff --git a/node_modules/react-native/React/AppSetup/RCTAppSetupUtils.mm b/node_modules/react-native/React/AppSetup/RCTAppSetupUtils.mm
index cc9641c..b44594a 100644
--- a/node_modules/react-native/React/AppSetup/RCTAppSetupUtils.mm
+++ b/node_modules/react-native/React/AppSetup/RCTAppSetupUtils.mm
@@ -23,33 +23,8 @@
 #import <React/RCTFabricSurfaceHostingProxyRootView.h>
 #endif
 
-#ifdef FB_SONARKIT_ENABLED
-#import <FlipperKit/FlipperClient.h>
-#import <FlipperKitLayoutPlugin/FlipperKitLayoutPlugin.h>
-#import <FlipperKitNetworkPlugin/FlipperKitNetworkPlugin.h>
-#import <FlipperKitReactPlugin/FlipperKitReactPlugin.h>
-#import <FlipperKitUserDefaultsPlugin/FKUserDefaultsPlugin.h>
-#import <SKIOSNetworkPlugin/SKIOSNetworkAdapter.h>
-
-static void InitializeFlipper(UIApplication *application)
-{
-  FlipperClient *client = [FlipperClient sharedClient];
-  SKDescriptorMapper *layoutDescriptorMapper = [[SKDescriptorMapper alloc] initWithDefaults];
-  [client addPlugin:[[FlipperKitLayoutPlugin alloc] initWithRootNode:application
-                                                withDescriptorMapper:layoutDescriptorMapper]];
-  [client addPlugin:[[FKUserDefaultsPlugin alloc] initWithSuiteName:nil]];
-  [client addPlugin:[FlipperKitReactPlugin new]];
-  [client addPlugin:[[FlipperKitNetworkPlugin alloc] initWithNetworkAdapter:[SKIOSNetworkAdapter new]]];
-  [client start];
-}
-#endif
-
 void RCTAppSetupPrepareApp(UIApplication *application)
 {
-#ifdef FB_SONARKIT_ENABLED
-  InitializeFlipper(application);
-#endif
-
 #if RCT_NEW_ARCH_ENABLED
   RCTEnableTurboModule(YES);
 #endif
