diff --git a/node_modules/react-native-blind-threshold-bls/android/build.gradle b/node_modules/react-native-blind-threshold-bls/android/build.gradle
index 637ee32..587ce9c 100644
--- a/node_modules/react-native-blind-threshold-bls/android/build.gradle
+++ b/node_modules/react-native-blind-threshold-bls/android/build.gradle
@@ -20,7 +20,7 @@ def safeExtGet(prop, fallback) {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 buildscript {
     // The Android Gradle plugin is only required when opening the android folder stand-alone.
@@ -39,7 +39,7 @@ buildscript {
 }
 
 apply plugin: 'com.android.library'
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
 
 android {
     compileSdkVersion safeExtGet('compileSdkVersion', DEFAULT_COMPILE_SDK_VERSION)
@@ -110,7 +110,7 @@ afterEvaluate { project ->
     task androidJavadoc(type: Javadoc) {
         source = android.sourceSets.main.java.srcDirs
         classpath += files(android.bootClasspath)
-        classpath += files(project.getConfigurations().getByName('compile').asList())
+        // classpath += files(project.getConfigurations().getByName('compile').asList())
         include '**/*.java'
     }
 
@@ -139,12 +139,12 @@ afterEvaluate { project ->
         archives androidJavadocJar
     }
 
-    task installArchives(type: Upload) {
-        configuration = configurations.archives
-        repositories.mavenDeployer {
-            // Deploy to react-native-event-bridge/maven, ready to publish to npm
-            repository url: "file://${projectDir}/../android/maven"
-            configureReactNativePom pom
-        }
-    }
+    // task installArchives(type: Upload) {
+    //     configuration = configurations.archives
+    //     repositories.mavenDeployer {
+    //         // Deploy to react-native-event-bridge/maven, ready to publish to npm
+    //         repository url: "file://${projectDir}/../android/maven"
+    //         configureReactNativePom pom
+    //     }
+    // }
 }
