diff --git a/node_modules/react-native-restart-android/android/build.gradle b/node_modules/react-native-restart-android/android/build.gradle
index f2b6cc2..54ee926 100644
--- a/node_modules/react-native-restart-android/android/build.gradle
+++ b/node_modules/react-native-restart-android/android/build.gradle
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
