diff --git a/node_modules/react-native-securerandom/RNSecureRandom.podspec b/node_modules/react-native-securerandom/RNSecureRandom.podspec
index aaae1a9..4bf80cc 100644
--- a/node_modules/react-native-securerandom/RNSecureRandom.podspec
+++ b/node_modules/react-native-securerandom/RNSecureRandom.podspec
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/rh389/react-native-securerandom.git", :tag => "master" }
   s.source_files = "ios/RNSecureRandom.{h,m}"
   s.requires_arc = true
-  s.dependency "React"
+  s.dependency "React-Core"
 end
