diff --git a/node_modules/react-native-exit-app/RNExitApp.podspec b/node_modules/react-native-exit-app/RNExitApp.podspec
index 7eed3fc..c0a7bfd 100644
--- a/node_modules/react-native-exit-app/RNExitApp.podspec
+++ b/node_modules/react-native-exit-app/RNExitApp.podspec
@@ -22,5 +22,5 @@ Pod::Spec.new do |s|
 	s.preserve_paths = 'README.md', 'package.json', '*.js'
 	s.source_files   = 'ios/RNExitApp/**/*.{h,m}'
 
-	s.dependency 'React'
+	s.dependency 'React-Core'
 end
