diff --git a/node_modules/react-native-shake/react-native-shake.podspec b/node_modules/react-native-shake/react-native-shake.podspec
index cc5975b..437bce7 100644
--- a/node_modules/react-native-shake/react-native-shake.podspec
+++ b/node_modules/react-native-shake/react-native-shake.podspec
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.source        = { :git => "https://github.com/Doko-Demo-Doa/react-native-shake", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m,swift}"
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
