diff --git a/node_modules/react-native-blind-threshold-bls/react-native-blind-threshold-bls.podspec b/node_modules/react-native-blind-threshold-bls/react-native-blind-threshold-bls.podspec
index 43fb326..08d92b6 100644
--- a/node_modules/react-native-blind-threshold-bls/react-native-blind-threshold-bls.podspec
+++ b/node_modules/react-native-blind-threshold-bls/react-native-blind-threshold-bls.podspec
@@ -19,7 +19,7 @@ Pod::Spec.new do |s|
   s.ios.vendored_library = 'ios/Libraries/libblind_threshold_bls.a'
   s.requires_arc = true
 
-  s.dependency "React"
+  s.dependency "React-Core"
   # ...
   # s.dependency "..."
 end
