diff --git a/node_modules/react-native-keep-awake/react-native-keep-awake.podspec b/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
index 95933a7..756ad5a 100644
--- a/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
+++ b/node_modules/react-native-keep-awake/react-native-keep-awake.podspec
@@ -18,5 +18,5 @@ Pod::Spec.new do |s|
   s.preserve_paths = 'README.md', 'package.json', 'index.js'
   s.source_files   = 'ios/*.{h,m}'
 
-  s.dependency 'React'
+  s.dependency 'React-Core'
 end
