diff --git a/node_modules/react-native-flag-secure-android/android/build.gradle b/node_modules/react-native-flag-secure-android/android/build.gradle
index 967a2d7..d6665c2 100644
--- a/node_modules/react-native-flag-secure-android/android/build.gradle
+++ b/node_modules/react-native-flag-secure-android/android/build.gradle
@@ -26,5 +26,5 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
