diff --git a/node_modules/react-native/Libraries/AppDelegate/RCTAppDelegate.h b/node_modules/react-native/Libraries/AppDelegate/RCTAppDelegate.h
index cf6cd3a..0e36e95 100644
--- a/node_modules/react-native/Libraries/AppDelegate/RCTAppDelegate.h
+++ b/node_modules/react-native/Libraries/AppDelegate/RCTAppDelegate.h
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+#import <UserNotifications/UserNotifications.h>
 #import <UIKit/UIKit.h>
 
 @class RCTBridge;
@@ -50,7 +51,7 @@
                                                          (const facebook::react::ObjCTurboModule::InitParams &)params
  *   - (id<RCTTurboModule>)getModuleInstanceFromClass:(Class)moduleClass
  */
-@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, UISceneDelegate, RCTBridgeDelegate>
+@interface RCTAppDelegate : UIResponder <UIApplicationDelegate, RCTBridgeDelegate, UNUserNotificationCenterDelegate>
 
 /// The window object, used to render the UViewControllers
 @property (nonatomic, strong) UIWindow *window;
