diff --git a/node_modules/react-native-sms/react-native-sms.podspec b/node_modules/react-native-sms/react-native-sms.podspec
index d1299f1..9eab342 100644
--- a/node_modules/react-native-sms/react-native-sms.podspec
+++ b/node_modules/react-native-sms/react-native-sms.podspec
@@ -12,6 +12,6 @@ Pod::Spec.new do |s|
    s.platform       = :ios, "8.0"
    s.source         = { :git => "#{package_json["repository"]["url"]}" }
    s.source_files   = 'SendSMS/*.{h,m}'
-   s.dependency 'React'
+   s.dependency 'React-Core'
 
 end
\ No newline at end of file
