diff --git a/node_modules/react-native-animated-to-reanimated/android/src/main/java/com/reactnativeanimatedtoreanimated/AnimatedToReanimatedViewManager.kt b/node_modules/react-native-animated-to-reanimated/android/src/main/java/com/reactnativeanimatedtoreanimated/AnimatedToReanimatedViewManager.kt
index d329c03..f86f566 100644
--- a/node_modules/react-native-animated-to-reanimated/android/src/main/java/com/reactnativeanimatedtoreanimated/AnimatedToReanimatedViewManager.kt
+++ b/node_modules/react-native-animated-to-reanimated/android/src/main/java/com/reactnativeanimatedtoreanimated/AnimatedToReanimatedViewManager.kt
@@ -26,7 +26,7 @@ class AnimatedToReanimatedViewManager : SimpleViewManager<View>() {
   @ReactProp(name = "value")
   fun setValue(view: View, value: Double) {
     val reactContext = view.context as ReactContext;
-    val dispatcher = reactContext.getNativeModule(UIManagerModule::class.java).eventDispatcher
+    val dispatcher = reactContext.getNativeModule(UIManagerModule::class.java)!!.eventDispatcher
 
     dispatcher.dispatchEvent(ValueChangeEvent(view.id, value))
   }
diff --git a/node_modules/react-native-animated-to-reanimated/ios/AnimatedToReanimatedViewManager.m b/node_modules/react-native-animated-to-reanimated/ios/AnimatedToReanimatedViewManager.m
index 2e42567..4b4b86c 100644
--- a/node_modules/react-native-animated-to-reanimated/ios/AnimatedToReanimatedViewManager.m
+++ b/node_modules/react-native-animated-to-reanimated/ios/AnimatedToReanimatedViewManager.m
@@ -1,29 +1,47 @@
 #import <React/RCTViewManager.h>
 
-@interface AnimatedToReanimatedViewManager : UIView
+@interface AnimatedToReanimatedViewManager : RCTViewManager
+
+@end
+
+@interface AnimatedToReanimatedView : UIView
 
 @property (nonatomic) NSNumber* value;
 @property (nonatomic, copy) RCTBubblingEventBlock onValueChange;
 
 @end
 
+@implementation AnimatedToReanimatedView
+
+- (void)setValue:(NSNumber*)value
+{
+  _value = value;
+    
+  if (!self.onValueChange) {
+    return;
+  }
+
+  // Dispatch async to avoid deadlock, this likely causes the
+  // listened value to be 1 frame behind
+  dispatch_async(dispatch_get_main_queue(), ^(void){
+    self.onValueChange(@{
+      @"value": (value ?: [NSNull null])
+    });
+  });
+}
+
+@end
+
 @implementation AnimatedToReanimatedViewManager
 
 RCT_EXPORT_MODULE(AnimatedToReanimatedView)
 
 - (UIView *)view
 {
-  return [[UIView alloc] init];
+  return [[AnimatedToReanimatedView alloc] init];
 }
 
 RCT_EXPORT_VIEW_PROPERTY(onValueChange, RCTBubblingEventBlock)
 RCT_EXPORT_VIEW_PROPERTY(value, NSNumber)
 
-- (void) setValue:(NSNumber*)value{
-  self.value = value;
-  self.onValueChange(@{
-    @"value": value
-  });
-}
-
 @end
