diff --git a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/FilterView.java b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/FilterView.java
index a3d9029..528f4d1 100644
--- a/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/FilterView.java
+++ b/node_modules/react-native-svg/android/src/main/java/com/horcrux/svg/FilterView.java
@@ -80,7 +80,8 @@ class FilterView extends DefinitionView {
 
     for (int i = 0; i < getChildCount(); i++) {
       View node = getChildAt(i);
-      if (node instanceof FilterPrimitiveView currentFilter) {
+      if (node instanceof FilterPrimitiveView) {
+        FilterPrimitiveView currentFilter = (FilterPrimitiveView) node;
         res = currentFilter.applyFilter(mResultsMap, res);
         String resultName = currentFilter.getResult();
         if (resultName != null) {
