diff --git a/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec b/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
index 8115b56..eb1ae7b 100644
--- a/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
+++ b/node_modules/react-native-fast-crypto/react-native-fast-crypto.podspec
@@ -30,5 +30,5 @@ Pod::Spec.new do |s|
   s.vendored_frameworks =
     "ios/secp256k1.xcframework"
 
-  s.dependency "React"
+  s.dependency "React-Core"
 end
